src/styles/search-results.css

```diff
@@ -0,0 +1,37 @@
+p {
+    text-align: center;
+    font-size: 20px;
+    color: black;
+}
+
+.search-results__image {
+    margin-top: 50px;
+    display: flex;
+    flex-direction: row; 
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    align-items: center;
```

Comment:
I dont think these properties are actually doing anything. If you remove them does the app still behave in the same way?